src/views/CoreUnitAbout/components/Markdown/renderUtils.tsx

```diff
@@ -2,15 +2,21 @@ import { styled } from '@mui/material';
 import React from 'react';
 import './Markdown.module.scss';
 
+// Use uuid to generate unique keys
+
+// Generate a stable key based on unique content rather than a random UUID
+const generateStableKey = (prefix: string, content: string | number) => `${prefix}-${content}`;
+
 export const customRenderer = {
   image(href: string) {
     return (
       <div
         style={{
           width: '100%',
         }}
+        key={generateStableKey('image', href)}
```

Comment:
This method could end up with duplicated keys when the same image is used twice so the href is the same. Maybe you can pass something like an index (if possible) to this render method so you can create a more unique key. Note that indexes are not recommended, but in this case, there's no sorting so it should not be an issue.

The same applies to the other render methods